x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_page/alerts_page.tsx

```diff
@@ -144,16 +144,14 @@ const AlertsPage: React.FunctionComponent = () => {
     pageSize: defaultPagination.pageSize,
     pageSizeOptions: [2, 5, 10, 20, 50, 100],
     leadingControlColumns: [],
-    renderCellValue: (rcvProps: EuiDataGridCellValueElementProps) => {
-      const { columnId, visibleRowIndex } = rcvProps as EuiDataGridCellValueElementProps & {
-        visibleRowIndex: number;
-      };
-      const value = (get(alerts[visibleRowIndex], columnId) ?? [])[0];
+    renderCellValue: ({ alert, field }: RenderCellValueProps) => {
+      const value = (get(alert, field) ?? [])[0];
```

Comment:
nitpick: I think we can just use `get`'s 3rd parameter, which is the default value

Reply:
👍